docs/README.md

```diff
@@ -4,34 +4,11 @@
 #### [Build and install cuML](../BUILD.md)
 
 #### Generate the docs
-```shell script
+```bash
 bash build.sh cppdocs pydocs
 ```
 
 #### Once the process finishes, documentation can be found in build/html
-```shell script
-xdg-open build/html/api.html`
-```
-
-## Building via Docker
-Pull or create the cuML [docker container](https://hub.docker.com/r/rapidsai/rapidsai/).
-
-### Start cuML container:
-```
-docker run -p 8000:8000 -it cuml bash
```

Comment:
I think this is referencing the container that we're removing in this PR. It seems like this method of building docs is not viable anymore, so I removed it.